config/locales/notices/en.yml

```diff
@@ -47,8 +47,7 @@ en:
     use_diff_email:
       link: use a different email address
       text_html: Or, %{link}
-  session_expired_html: Please %{link} the page to log in.
-  session_expired_link: refresh
   session_timedout: >
     We signed you out. For your security, %{app} automatically ends your session
     when you haven’t moved to a new page for %{minutes} minutes.
+  session_cleared: We cleared your information for your security.
```

Comment:
this should be within the `notices` key (the top-level namespace for each config/locale file is the same as the file name). Looks like we have a few that are not conforming to this organization but in general that is how all of our locales are organized.